lib/conversation-manager.js

```
@@ -0,0 +1,429 @@
+  /**
+   * Check for new comments in a thread
+   */
+  async checkForNewComments(prNumber, originalCommentId, since) {
+    // Get all comments on the PR
+    const comments = await this.provider.getComments(prNumber);
+    
+    // Filter for replies to our comment that are newer than 'since'
+    return comments.filter(c => {
+      const isReply = c.in_reply_to_id === originalCommentId || 
+                     (c.body && c.body.includes(`@${this.provider.currentUser}`));
+      const isNewer = !since || new Date(c.created_at) > since;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)